CONTRIBUTING.md

```diff
@@ -0,0 +1,48 @@
+# Contributing to ESTELA
+We love your input! We want to make contributing to this project as easy and transparent as possible, whether it's:
+
+- Reporting a bug
+- Discussing the current state of the code
+- Submitting a fix
+- Proposing new features
+- Becoming a maintainer
+
+## We Use [Github Flow](https://github.com/bitmakerla/estela/actions), So All Code Changes Happen Through Pull Requests
+Pull requests are the best way to propose changes to the codebase. We actively welcome your pull requests:
+
+1. Fork the repo and create your branch from `master`.
+2. If you've added code that should be tested, add tests.
```

Comment:
do We have a structure for tests? 
In what file or folder the contributors add the tests?
What is the approach to make test?
This and other questions needs to be answered.